tests/test_buku.py

```diff
@@ -10,7 +10,8 @@
 
 from buku import is_int, parse_tags
 
-only_python_3_5 = pytest.mark.skipif(sys.version_info < (3, 5), reason="requires python3.5")
+only_python_3_5 = pytest.mark.skipif(
+    sys.version_info < (3, 5), reason="requires minimal python3.5")
```

Comment:
"requires minimal python3.5"

should be 

    "requires Python 3.5 or later"